### PR TITLE
feat(replay): viewed_by_id migration

### PR DESCRIPTION
**Pull request by aliu3ntry**

### CODE REVIEW

File: snuba/snuba_migrations/replays/0018_add_viewed_by_id_column.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        yield operations.DropColumn(
+            StorageSetKey.REPLAYS,
+            "replays_local",
+            column.name,
+            operations.OperationTarget.LOCAL,
+        )
+
+
+columns: List[Tuple[str, Column[Modifiers]]] = [
+    ("user_id", Column("viewed_by_id", UInt(64)))
```
Comment (cmanallen) [inline snuba/snuba_migrations/replays/0018_add_viewed_by_id_column.py:56]: Lets place it after `user_email`.  Just to keep the groups separated.

**Review by cmanallen — CHANGES_REQUESTED** [review]
Need to update `group_loader.py`.

**Review by cmanallen — APPROVED** [review]

File: snuba/snuba_migrations/replays/0018_add_viewed_by_id_column.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        yield operations.DropColumn(
+            StorageSetKey.REPLAYS,
+            "replays_local",
+            column.name,
+            operations.OperationTarget.LOCAL,
+        )
+
+
+columns: List[Tuple[str, Column[Modifiers]]] = [
+    ("user_email", Column("viewed_by_id", UInt(64)))
```
Comment (volokluev) [inline snuba/snuba_migrations/replays/0018_add_viewed_by_id_column.py:56]: what does `user_email` have to do with anything here?

Comment (aliu3ntry) [reply snuba/snuba_migrations/replays/0018_add_viewed_by_id_column.py:56]: This is the column that it comes after

Comment (aliu3ntry) [reply snuba/snuba_migrations/replays/0018_add_viewed_by_id_column.py:56]: Following the boilerplate we used in 0017

**Comment by volokluev** [comment]
This looks good to me. You can make the check pass by moving the yaml changes into a different PR. This makes sure the column will be there before we start being able to query it

**Review by volokluev — APPROVED** [review]

**Comment by aliu3ntry** [comment]
yaml changes moved to https://github.com/getsentry/snuba/pull/5709

**Pull request merged**